dashboard/api_test.py

```diff
@@ -252,15 +255,15 @@ def setUpTestData(cls):
             cls.enrollments_json = json.loads(file_obj.read())
 
         cls.enrollments = Enrollments(cls.enrollments_json)
+        cls.now = datetime.now(pytz.utc)
 
     def test_status_for_run_not_enrolled(self):
         """test for get_status_for_courserun"""
```

Comment:
I just noticed that all of these functions have the same docstring... it's outside the scope of this PR but would you mind updating them?